### ISSUE TITLE
Unable to uninstall

### ISSUE THREAD

**Issue by allanmcpherson** [anbox/anbox#753]
When I installed Anbox, I must have misread the memory requirements. It's taking up half my SSD, which is way too much to give up for what I wanted it for. I have seen a solution to a similar problem, but it didn't help. It doesn't show as installed anymore, but it's still taking up half the disk.

I can't seem to remove and of the files or directories associated with it, and I can't even see the partition of the disk that it took to reformat it. I'm running Ubuntu 16.04.

I would greatly appreciate any help getting my disk space back.

**Comment by morphis** [comment]
How big is your SSD? The compressed snap doesn't take more than ~370MB and the data Android creates at runtime is without any additional installed application around 40MB

Can you please describe in detail how you installed Anbox, which additional applications you installed etc.

Anbox is not taking any partition. It comes as a snap which a squashfs formatted file which gets loop-mounted onto /snap/anbox/current/.The file itself never gets unpacked and decompression happens on the fly. You find the snap files at `/var/lib/snapd/snaps/`:

```
$ ls -alh /var/lib/snapd/snaps/anbox_104.snap 
-rw-r--r-- 2 root root 372M Jun 10 11:59 /var/lib/snapd/snaps/anbox_104.snap
$ mount | grep anbox
/var/lib/snapd/snaps/anbox_104.snap on /snap/anbox/104 type squashfs (ro,nodev,relatime,x-gdu.hide)
$ sudo du -hs /var/snap/anbox
36M	/var/snap/anbox
```
So in summary Anbox will not take more than ~400MB after the initial installation. Any additional space will be taken by manually installed applications.

If you followed the std. installation instructions (installation as a snap) then you find instructions of how to remove Anbox at https://github.com/anbox/anbox/blob/master/docs/install.md#uninstall-anbox

**Comment by allanmcpherson** [comment]
I have a 512 GB drive, so I was surprised when it seemed to have eaten up a
huge amount of space. It was showing up as two sizeable mounted partitions.
I had followed the instructions given on the website to install it, and I
hadn't actually gotten as far as installing any applications. I've followed
the instructions you provided to uninstall it, and it seems to have
released the space.

Thanks
